sdk/src/main/java/com/horizen/box/data/ForgerBoxData.java

```diff
@@ -79,25 +79,4 @@ public static ForgerBoxData parseBytes(byte[] bytes) {
     public byte[] customFieldsHash() {
         return Blake2b256.hash(Bytes.concat(rewardProposition().pubKeyBytes(), vrfPublicKey().key()));
     }
-
-    @Override
```

Comment:
I prefer to use `public boolean equals(Object obj) {super.equals(obj);}` in such cases, just for mention that it was done by reason (with appropriate comment).